src/components/studies/app-design/AppDesign.tsx

```diff
@@ -316,6 +321,80 @@ const AppDesign: React.FunctionComponent<
     isIrbEmailValid: true,
   })
 
+  const [errorState, setErrorState] = React.useState({
+    studyTitleHasError: false,
+    studySummaryCopyHasError: false,
+    leadPINameHasError: false,
+    leadPIAffiliationHasError: false,
+    contactLeadNameHasError: false,
+    contactLeadPositonHasError: false,
+    irbRecordNameHasError: false,
+    irbProtocolIdHasError: false,
+  })
+
+  useEffect(() => {
+    if (!showError) return
+    const updatedErrorState = {
+      studyTitleHasError: false,
+      studySummaryCopyHasError: false,
+      leadPINameHasError: false,
+      leadPIAffiliationHasError: false,
+      contactLeadNameHasError: false,
+      contactLeadPositonHasError: false,
+      irbRecordNameHasError: false,
+      irbProtocolIdHasError: false,
+    }
+    const contactLead = study.contacts?.find(el => el.role === 'study_support')
+    const principalInvestigator = study.contacts?.find(
```

Comment:
is there a way to simplify/shorten this?  May be build an object that correlates the error with the property/value? 
It seems like you are applying similar logic to different properties There should be some way to take advantage of repeating pattern